packages/client/src/pages/FindAMentor/MentorSearch.css

```diff
@@ -0,0 +1,95 @@
+.ais-SearchBox {
```

Comment:
Are we doing css files?

Reply:
@oupadhyay will remove this an upcoming CL